spec/lib/simp/cli/config/items/action/set_hostname_action_spec.rb

```diff
@@ -11,7 +11,25 @@
   # TODO:  test successes with acceptance tests
   describe "#apply" do
     it "will do set hostname " do
-      skip "FIXME: how shall we test SetHostnameAction#apply()?"
+      skip("Test can't be run as root") if ENV['USER'] == 'root' or ENV['HOME'] == '/root'
```

Comment:
Is this skip necessary since you have mocked out the system calls?

Reply:
@lnemsick-simp Probably not but maybe?